packages/appstore/src/types/common.types.ts

```diff
@@ -78,3 +78,22 @@ export type TStaticAccountProps = {
     platform: TPlatform;
     type: TMarketType;
 };
+
+export type TIconTypes =
+    | 'Derived'
+    | 'Financial'
+    | 'BinaryBot'
+    | 'BinaryBotBlue'
+    | 'DBot'
+    | 'Demo'
+    | 'DerivGo'
+    | 'DerivGoBlack'
+    | 'DerivLogo'
+    | 'DerivTradingLogo'
+    | 'DerivX'
+    | 'DropDown'
+    | 'DTrader'
+    | 'Options'
+    | 'SmartTrader'
+    | 'SmartTraderBlue'
+    | 'CFDs';
```

Comment:
Why do we need all these types?

Reply:
Since we are using 'TradigPlatformIconProps' we need to use its type for available icons.

Reply:
@hirad-rewok you can use these types from `packages/appstore/src/assets/svgs/trading-platform/index.tsx`